### DISCUSSION TITLE
Exporting hierarchy of reference namespace

### DISCUSSION THREAD

**Discussion by alvaccariello** [geovistory/.github#22] [Ideas]
When exporting a project in RDF or OWL-DL format, the hierarchy of the project created on Ontome does not include the hierarchy of the classes belonging to reference namespace. Is there a way to export a project created on Ontome, preserving all the parental relations deriving from reference namespace hierarchies?

**Comment by valamercery** [comment]
Thanks for your feedback. As this feature is not implemented yet in OntoME, I'm changing the category of this discussion from q-and-a to ideas.

**Comment by alvaccariello** [comment]
Dear Vincent,
Thank you for that. I was not sure about the proper category in the first place.
See you in Paris!
Best regards,
Alessia
